src/Tasks/Microsoft.NET.Build.Tasks/ResolveFrameworkReferences.cs

```diff
@@ -18,6 +19,8 @@ namespace Microsoft.NET.Build.Tasks
     /// </summary>
     public class ResolveFrameworkReferences : TaskBase
     {
+        private const string WindowsDesktopFramework = "Microsoft.WindowsDesktop.App";
```

Comment:
Instead of hard-coding the name here, I'd suggest adding metadata to the KnownFrameworkReference specifying that it's only supported on Windows.  Until that change makes it through core-sdk, you can add it in the ResolveFrameworkReferences target:

```xml
<KnownFrameworkReference Update="Microsoft.WindowsDesktop.App" IsWindowsOnly="True" />
<KnownFrameworkReference Update="Microsoft.WindowsDesktop.App.WindowsForms" IsWindowsOnly="True" />
<KnownFrameworkReference Update="Microsoft.WindowsDesktop.App.WPF" IsWindowsOnly="True" />
```

Reply:
Great idea.  I'll make that change.